asset_dashboard/models.py

```diff
@@ -37,29 +37,41 @@ class Project(models.Model):
     commissioner_districts = models.ManyToManyField('CommissionerDistrict', blank=True)
     zones = models.ManyToManyField('Zone', blank=True)
 
-    PHASE_CHOICES = [
+    def __str__(self):
+        return self.name
+
+
+class Task(models.Model):
+    """
+    A sub-project unit of work. Projects without defined tasks are assigned a
+    default Task of type "implementation".
+    """
+
+    # TODO: Identify better choice list (e.g., what's the difference between
```

Comment:
We'll need to rope Garret in for this, I think. Can address separately.

Reply:
okay we are going to go with 

Feasibilty
Design
Engineering
Construction,

Implementation,

we are going to do away with reference to phase here.